common/sku_customization.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -ex
+
+# setup systemd service
+cat <<EOF >/usr/sbin/setup_topo.sh
```

Comment:
Should we rename this to something link - "setup_sku_customizations". In future, this service can be used to add other SKU specific customizations.

Reply:
Changed the filename